### PR TITLE
Added Notebook created by Jerom

### PR DESCRIPTION
**Pull request by jeromaerts**
lorentz workshop comparison file

### CODE REVIEW

File: Lorentz_model_comparison.ipynb [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 1.61 ms, sys: 354 µs, total: 1.97 ms\n",
+      "Wall time: 1.08 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Read the .ini file and initialize the model\n",
+    "%time pcrg.initialize(\"pcrg_rhinemeuse.ini\")"
```
Comment (nielsdrost) [inline Lorentz_model_comparison.ipynb:95]: We should insert usage of parameter db here. Manually added it, works.

File: Lorentz_model_comparison.ipynb [hunk 2]
```diff
@@ -0,0 +1,606 @@
...
+    "import datetime\n",
+    "from datetime import datetime\n",
+    "from datetime import timedelta\n",
+    "\n",
+    "import numpy\n",
+    "import numpy.ma as ma\n",
+    "import pandas as pd\n",
+    "import xarray as xr\n",
+    "\n",
+    "import hydrostats.data as hd\n",
```
Comment (nielsdrost) [inline Lorentz_model_comparison.ipynb:38]: Hydrostats not available in lab environment

File: Lorentz_model_comparison.ipynb [hunk 3]
```diff
@@ -0,0 +1,606 @@
...
...ULqVOXbp0CXv37kVxcTGaNm2KgIAApUuqE9euXYO9vT3s7Oyg1Wphbm6udEl1gvNpWjifpoXzaXhsiIlq4MKFC+jcuTOsra0RExODXr16KV1SnUhOToafnx8GDx6MGzdu4ObNm7C1tcX27dvh6ekJSZKULrFWnD9/Hn379kWHDh3w008/wdHRUfFv0nWB88n5NEacT85nXWJDTFRNN2/exPTp02FjY4Pi4mL8+uuv2L9/P7y9vZUurVbdvn0bI0aMwODBg7Fq1SoUFxfj2LFjGDp0KHr16oX169ejT58+EEIY9TfqjIwMPP/889DpdCgvL4eVlRX27t2r+Dfp2sb55HwaI84n57OucQ0xUTVdvnwZbm5uCAwMxLZt2zBw4EA8+eST8rPcTeVnzZycHNy5cwdTp04FAFhbW6Nr167w9vZGVlYWpk+fjpKSEkiSZNSZ4+LiYGVlhffffx/z589HaWkpxo8fj1u3bsHc3Bzl5eVKl1grOJ+cT2PE+eR81jU2xETV1LdvX7z++usYOXIkGjVqhM8++wy+vr4YPXo0EhIS5J/ehRAoKSlRuNrqs7S0RF5eHmJjY+Vjt27dQmFhIT7//HOUlZVh3rx5AGDUdyymTJmCkJAQDBo0CE8//TRCQ0Plb9I5OTmwsLCoF2/8qCnOJ+fTGHE+OZ91jY9uJqqB5s2byx/b2NhgxIgRSEpKwvvvv48hQ4agefPmeO+995CVlQUvLy+j/AYmhEBKSgri4uKQmJiIW7duYfz48Rg3bhzeeOMNpKenIzU1Fc8//7zSpdZY+/bt5Y87dOgAFxcXHDlyBLt27cKECRNgY2OD8PBwtGrVCvb29gpWWn2cT86nseJ8cj7rEtcQEz2iK1eu4OjRo0hLS8OTTz553zfQ5eTk4LXXXkN8fDz69++Pb775BsnJyejcubOBK66eK1eu4Pjx47hz5w769OmDDh064H//+x/WrVuH//73v9BoNBg3bhwqfpZetGgRDh06hKNHjxrVOr6srCykpKSgoKAAvXr1gqurq7x2rWJ9nlarxb59+7B8+XJYW1ujffv22LhxIy5cuKD3zbw+43xyPjmf9Rfnsx7NpyCihzp16pRo27at6Nevn/Dw8BANGjQQe/fuFUIIodVqK42/fv26cHFxEY0bNxYnTpwwdLnVdvLkSdG0aVPRuXNn0apVK2FtbS2WLVs
```
Comment (nielsdrost) [inline Lorentz_model_comparison.ipynb:587]: Should be Python 3

**Comment by nielsdrost** [comment]
He @sverhoeven could you have a look?

File: Lorentz_model_comparison.ipynb [hunk 4]
```diff
@@ -0,0 +1,606 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Import docker and set client\n",
+    "import docker\n",
+    "client = docker.from_env()"
```
Comment (sverhoeven) [inline Lorentz_model_comparison.ipynb:18]: Please remove cell as client var is not used in notebook.

File: Lorentz_model_comparison.ipynb (hunk 2 above)
Comment (sverhoeven) [reply Lorentz_model_comparison.ipynb:38]: It is now available in the lab environment, but the import is not used anywhere. 
The import should be removed or be used.

File: Lorentz_model_comparison.ipynb (hunk 1 above)
Comment (sverhoeven) [reply Lorentz_model_comparison.ipynb:95]: Agree, otherwise we are unable to run it.

File: Lorentz_model_comparison.ipynb [hunk 5]
```diff
@@ -0,0 +1,606 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 165,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import GRDC station data, set index\n",
+    "obs = dataframe_from_stationId('./validation/', 6435060)\n",
```
Comment (sverhoeven) [inline Lorentz_model_comparison.ipynb:494]: Similar as model config+dataset, the validation file appears out of nowhere. Please provide instructions how to get it.

**Review by sverhoeven — CHANGES_REQUESTED** [review]

**Review by sverhoeven — APPROVED** [review]

**Pull request merged**